.changeset/pink-laws-smell.md

```diff
@@ -0,0 +1,5 @@
+---
+"pnpm": major
+---
+
+Side effects cache is turned on by default. To turn it off, use `side-effects-cache=true`.
```

Comment:
@zkochan Is this a typo? Otherwise it sounds very confusing —setting `feature=true` to _disable_ said feature, I mean.

Reply:
Yes, it is a mistake.